.travis.yml

```diff
@@ -7,9 +7,15 @@ php:
   - 5.6
   - hhvm
 
+matrix:
+  include:
+    - php: 5.6
+      env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable'
+  fast_finish: true
+
 before_script:
   - composer self-update
-  - composer install --dev --prefer-source
+  - composer update $COMPOSER_FLAGS --prefer-source -n
```

Comment:
Ah, nevermind, I did not know that. Are they using an environment variable or something?